doc/1/plugins/plugin-context/customize-errors/index.md

```diff
@@ -0,0 +1,61 @@
+---
+code: true
+type: page
+title: customize errors
+---
+
+# Customize errors
+
+When creating your own Kuzzle plugin, you can use the `errorsManager` to have customizable errors.
+
+In order to define it, you have to write them inside the [manifest.json](https://docs.kuzzle.io/core/1/plugins/guides/manual-setup/prerequisites/#manifest-json) in a `errors` field.
+
+Your manifest would be something like :
+```
+{
+    "name": "kuzzle-plugin-xxx",
+    "kuzzleVersion": ">=1.0.0 <2.0.0",
+    "privileged": false,
+    "errors": {
+        "some_error": {
+            "code": 1,
+            "message": "Some error occurred: %s",
+            "class": "BadRequestError"
+	},
+        "some_other_error": {
+            "code": 2,
+            "message": "Some other error occurred: %s",
+            "class": "ForbiddenError"
+	}
+    }
+}
+```
+
+It is exposed as `errorsManager` in the [PluginContext](https://docs.kuzzle.io/core/1/plugins/plugin-context/accessors/intro/).
+The `errorsManager` provides two functions:
+- To throw : `context.errorsManager.throw(error, placeholders);`.
+- To get the built error: `context.errorsManager.getError(error, placeholders);`
```

Comment:
The `getError` method has to be renamed in `get` since `errorsManager.getError` is a little bit redundant.